serverless-plugin/index.js

```diff
@@ -49,6 +49,9 @@ class ServerlessPlugin {
             haveImage: false,
         };
 
+        // Do not try to look into Node dependencies
+        this.serverless.service.package.excludeDevDependencies = false;
```

Comment:
I don't understand this. If `excludeDevDependencies` is `false` then surely that means they  would be _included_. But the comment say "Do _not_ try to look into node dependencies".